test/hooks.on-listen.test.js

```diff
@@ -0,0 +1,614 @@
+'use strict'
+
+const { t, before } = require('tap')
```

Comment:
nitpick:
We usually import test and not t, then you dont need to do t.test, but can directly do test()
```suggestion
const { test, before } = require('tap')
```

Reply:
We were using `test\hooks.on-ready.test.js` as a reference which uses 't'. It also uses t.pass() in a similar fashion for hooks.

Reply:
In this case, you have to expect that on-ready.test.js is very old. So this is kind of historical. You can keep the t.test notation, if you like. Maybe this is just personal preference.